src/webob/acceptparse.py

```diff
@@ -408,6 +408,24 @@ def generator(value):
                 )
         return generator(value=value)
 
+    def _parse_and_normalize_offers(self, offers):
```

Comment:
Could I ask what "normalize" means here?

Reply:
mostly just lowercase

Reply:
but also guarantee that everything that is returned passes the media type abnf, so the function is doing parse/filter/lowercase.

Reply:
Normalize in computing tends to be about making sure that input is valid and is in a consistent form so that further operations can rely on the data being "clean".

See https://en.wikipedia.org/wiki/Text_normalization